frontend/src/components/menuDesktop/MenuDesktop.tsx

```diff
@@ -16,51 +16,59 @@ const icons = [
 ];
 
 export const Menu = () => {
+  const location = useLocation();
+  const getPath = (input: string) => input.split(' ').join('').toLowerCase();
   return (
     <>
       <div className={styles.logo_div}>
         <img className={styles.logo_menu} src='/assets/icons/logo-menu.svg' alt='Logo' />
         <h4>Dashhhboard</h4>
-        <img className={styles.logo_menu} src='/assets/icons/plus.svg' alt='Plus Icon' />
       </div>
       <List className={styles.list_top}>
         {['Dashboard', 'Project', 'My Task', 'Activity', 'Team', 'Message', 'Setting'].map(
           (text, index) => (
-            <ListItem key={text} disablePadding className={styles.list_item}>
-              <ListItemButton>
-                <Link
-                  className={styles.list_item_link}
-                  to={index === 0 ? '/' : `/${text.split(' ').join('').toLowerCase()}`}
-                >
-                  <ListItemIcon>
+            <Link
+              key={text}
+              className={styles.list_item_link}
+              to={index === 0 ? '/' : `/${getPath(text)}`}
+            >
+              <ListItem
+                disablePadding
+                className={`${styles.list_item} ${
+                  (location.pathname === '/' && text === 'Dashboard') ||
+                  location.pathname.includes(getPath(text))
+                    ? styles.current_path
+                    : ''
+                }`}
+              >
```

Comment:
not loving this code, it is quite difficult to understand what is going on, but I don't have any suggestions to how you could change this.